fix: add missing info about docs/config.json (#429)
exercism/docs: building/tracks/docs.md

├── SNIPPET.txt (required)
├── REPRESENTER_NORMALIZATIONS.md (optional)
├── RESOURCES.md (required)
├── TESTS.md (required)

> └── config.json (required)

Reviewer: Technically, this file is not required (as in: nothing breaks when it isn't there). I'd be happy to list it here as required though, as it is usually what one wants.

Author: Just made it optional! If it's optional, it's optional!

Reviewer 2: Is it the `TESTS.md` that's optional or `config.json`?

Author: @thinkverse The whole reason this PR was started in the first place was because [V's config.json was missing, causing breakage](https://forum.exercism.org/t/v-missing-track-specific-documentation/3402) so I'm guessing `TESTS.md`

> ├── TESTS.md (optional)
> └── config.json (required)

Reviewer: ```suggestion
├── TESTS.md (required)
└── config.json (required)
```

They're both required

Author: Oh sorry, then which file were you referring to [here](https://github.com/exercism/docs/pull/429#discussion_r1162383332)?

Reviewer: The `config.json` file

Author: So `config.json` is not required, but causes breakage down the line if it's not included? Seems odd to me, but not blocking so :man_shrugging:

Check [this page](https://exercism.org/docs/tracks/fsharp/resources) to see what this looks like when rendered.

> 

Reviewer 2: ```suggestion
```

````

Check [this page](https://exercism.org/docs/tracks/fsharp/tests) to see what this looks like when rendered.

## File: `config.json`

**Purpose:** Tell the website what pages are available.

**Displayed on:** (not displayed)

### Example

````json

> {
>   "docs": [
>     {
>       "uuid": "a2d18097-4e65-4826-85b9-4ed8f71689bd",
>       "slug": "installation",
>       "path": "docs/INSTALLATION.md",
>       "title": "Installing V locally",
>       "blurb": "Learn how to install V locally to solve Exercism's exercises on your own machine"
>     },
>     {
>       "uuid": "afa98f6a-9af2-4fc0-b9a6-b9f3df2289ab",
>       "slug": "learning",
>       "path": "docs/LEARNING.md",
>       "title": "How to learn V",
>       "blurb": "An overview of how to get started from scratch with V"
>     },
>     {
>       "uuid": "038fbf82-4c35-4b43-a30d-d73cda5b4c31",
>       "slug": "tests",
>       "path": "docs/TESTS.md",
>       "title": "Testing on the V track",
>       "blurb": "Learn how to test your V exercises on Exercism"
>     },
>     {
>       "uuid": "98eaa5bb-f4a2-4cf0-b92e-abe0e62068cd",
>       "slug": "resources",
>       "path": "docs/RESOURCES.md",
>       "title": "Useful V resources",
>       "blurb": "A collection of useful resources to help you master V"
>     }
>   ]
> }

Reviewer 2: Given most of the document uses examples from the F# track, it would make more sense to continue with that instead of using the V track as an example here. Thoughts @iHiD?

```suggestion
{
  "docs": [
    {
      "uuid": "410d95ba-6294-4f40-af4a-6bb052977a3e",
      "slug": "installation",
      "path": "docs/INSTALLATION.md",
      "title": "Installing F# locally",
      "blurb": "Learn how to install F# locally to solve Exercism's exercises on your own machine"
    },
    {
      "uuid": "e0862775-d757-4d21-826f-b7cc2640ae8f",
      "slug": "learning",
      "path": "docs/LEARNING.md",
      "title": "How to learn F#",
      "blurb": "An overview of how to get started from scratch with F#"
    },
    {
      "uuid": "bca11b18-be81-468a-a34e-f5e6ac16bfd6",
      "slug": "tests",
      "path": "docs/TESTS.md",
      "title": "Testing on the F# track",
      "blurb": "Learn how to test your F# exercises on Exercism"
    },
    {
      "uuid": "39a578fe-e4ae-479e-b648-733882e244ef",
      "slug": "resources",
      "path": "docs/RESOURCES.md",
      "title": "Useful F# resources",
      "blurb": "A collection of useful resources to help you master F#"
    },
    {
      "uuid": "edfbb0fa-a950-4384-899f-6c6c7a440dca",
      "slug": "representer-normalizations",
      "path": "docs/REPRESENTER_NORMALIZATIONS.md",
      "title": "F# representer normalizations",
      "blurb": "An overview of the normalizations the F# representer applies to solutions"
    }
  ]
}
```

Reviewer: Yep, I agree with that suggestion!

````

Check [this page](https://exercism.org/docs/tracks/fsharp/tests) to see what this looks like when rendered.

## File: `config.json`

**Purpose:** Tell the website what pages are available.

**Displayed on:** (not displayed)

### Example

> ````json

Reviewer 3: ```suggestion
```json
```

Prefer to use the minimum number of backticks for a code fence.

There are 4 backticks on lines 294/318 only because that code fence contains inner code blocks that use 3 backticks.

      "path": "docs/INSTALLATION.md",
      "title": "Installing F# locally",
      "blurb": "Learn how to install F# locally to solve Exercism's exercises on your own machine"
    },
    {
      "uuid": "e0862775-d757-4d21-826f-b7cc2640ae8f",
      "slug": "learning",
      "path": "docs/LEARNING.md",
      "title": "How to learn F#",
      "blurb": "An overview of how to get started from scratch with F#"
    },
    {
      "uuid": "bca11b18-be81-468a-a34e-f5e6ac16bfd6",
      "slug": "tests",
      "path": "docs/TESTS.md",
      "title": "Testing on the F# track",
      "blurb": "Learn how to test your F# exercises on Exercism"
    },
    {
      "uuid": "39a578fe-e4ae-479e-b648-733882e244ef",
      "slug": "resources",
      "path": "docs/RESOURCES.md",
      "title": "Useful F# resources",
      "blurb": "A collection of useful resources to help you master F#"
    },
    {
      "uuid": "edfbb0fa-a950-4384-899f-6c6c7a440dca",
      "slug": "representer-normalizations",
      "path": "docs/REPRESENTER_NORMALIZATIONS.md",
      "title": "F# representer normalizations",
      "blurb": "An overview of the normalizations the F# representer applies to solutions"
    }
  ]
}

> ````

Reviewer 3: ````suggestion
```
````

````

Check [this page](https://exercism.org/docs/tracks/fsharp/tests) to see what this looks like when rendered.

> 

Reviewer 2: ```suggestion
```

## File: `config.json`

> **Purpose:** Tell the site builder what pages are available.

Reviewer 4: ```suggestion
**Purpose:** Tell the website what pages are available.
```
